department-of-veterans-affairs/caseflow
# Added filtering to the table based on claim evidence results.

Resolves [[Create document filtering logic on Reader frontend](https://jira.devops.va.gov/browse/APPEALS-31880)

# Description
The results of the returned documents are now displayed and filtered in the documents table. Currently, results are faked.

## Acceptance Criteria

## Testing Plan

1. Log into caseflow as a user who can access Reader (BVAAABSHIRE works fine)
2. Navigate to reader
3. Click the Search button with no text in the input. This will fake results being successfully retrieved. 
4. Notice that there is now a list of filtered documents. We are returning all of them so there is no actual reduction of documents.
5. Click the 'Clear All Filters' text/button. Notice that the filter has been undone.
6. Enter some text into the search bar that isn't just spaces and click the search button again. This time, the backend has faked no data return, and since there were no matches, the filter was ignored.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="1306" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/53dde6e2-d64e-41f8-a856-b706e1fd5950">
|
<img width="1326" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/b6f4faaf-d825-41f9-889e-18867ba40774">
